sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatRequestAssistantMessage.java

```diff
@@ -214,8 +252,14 @@ public static ChatRequestAssistantMessage fromJson(JsonReader jsonReader) throws
                     reader.skipChildren();
                 }
             }
-            ChatRequestAssistantMessage deserializedChatRequestAssistantMessage
-                = new ChatRequestAssistantMessage(content);
+            ChatRequestAssistantMessage deserializedChatRequestAssistantMessage;
+            if (CoreUtils.isNullOrEmpty(stringContent) && chatMessageContentItems == null) {
+                deserializedChatRequestAssistantMessage = new ChatRequestAssistantMessage(content);
```

Comment:
we should either set `content` value when we deserialize the JSON to object, setting `stringContent` or `chatMessageContentItems`. Otherwise, content is null and new ChatRequestAssistantMessage(content=null), then the user calls getContent returns null (which is the reason of hide `getContent` will take off effort to maintain BinaryData `content`.